### PR TITLE
Zoey_Completed dom-querying

### PR DESCRIPTION
**Pull request by Zoey1318**
dom

### CODE REVIEW

File: dom-querying/main.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+console.log('hello, world');
+
+document.querySelector('h1');
+console.log(document.querySelector('h1'));
+console.dir('h1');
+
+document.querySelector('p#explanation');
+console.log(document.querySelector('p#explanation'));
+console.dir('p#explanation');
```
Comment (scrumptiousdev) [inline dom-querying/main.js:9]: So for `console.dir()`s, we want to actually console dir the value of what we got from `querySelector`

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]

**Comment by Zoey1318** [comment]
dom-fixed commented code

**Comment by Zoey1318** [comment]
sorry, I commited and realized I did wrong.

**Comment by scrumptiousdev** [comment]
> sorry, I commited and realized I did wrong.

No worries, Zoey! I know the instructions were very unclear about what you need to do. I will have that fixed!

**Comment by Zoey1318** [comment]
dom-fixed

**Review by scrumptiousdev — APPROVED** [review]
🪄 Approved!

**Pull request merged**